jujugui/static/gui/src/app/components/added-services-list/item/item.js

```diff
@@ -107,29 +101,20 @@ class AddedServicesListItem extends React.Component {
         onMouseEnter={this._onMouseEnter.bind(this)}
         onMouseLeave={this._onMouseLeave.bind(this)}
         role="button"
-        tabIndex="0">
+        tabIndex="0"
+      >
         <img className="inspector-view__item-icon" src={service.icon} />
-        <span className="inspector-view__item-count">
-          {service.unit_count}
-        </span>
-        {' '}
-        <span className="inspector-view__item-name">
-          {service.name}
-        </span>
-        <span className="inspector-view__status-block">
-          {statusIndicator}
-        </span>
+        <span className="inspector-view__item-count">{service.unit_count}</span>{' '}
+        <span className="inspector-view__item-name">{service.name}</span>
+        <span className="inspector-view__status-block">{statusIndicator}</span>
       </li>
```

Comment:
This does look better to me; I like where possible having a span be a single line.